interface/modules/zend_modules/module/Carecoordination/src/Carecoordination/Model/EncounterccdadispatchTable.php

```diff
@@ -18,6 +18,7 @@
 use Application\Listener\Listener;
 use Application\Model\ApplicationTable;
 use Carecoordination\Model\CarecoordinationTable;
+use Couchbase\SearchQuery;
```

Comment:
Just realized this was here, not sure how it got in here.  I'll remove it tomorrow.